src/api/web/index.ts

```diff
@@ -78,6 +78,24 @@ export default function(config, auth, storage) {
     });
   }
 
+  // Favicon
+  const customFaviconPath = _.get(config, 'web.favicon') || '';
+  router.get('/-/mapped/favicon', function(req, res): void {
+    if (customFaviconPath) {
```

Comment:
Here might check whether path is absolute, validation step. We never know what users will intent do add to `favicon` property. https://nodejs.org/api/path.html#path_path_isabsolute_path

We might ignore those are not absolute and return the default one.